app/screens/ServerView.tsx

```diff
@@ -180,6 +192,12 @@ const ServerView: React.FC<{
           value={password}
           onChangeText={setPassword}
         />
+        <SettingsSwitch
+          title="Force plain text password"
+          subtitle={usePlainPassword ? 'on' : 'off'}
```

Comment:
Can you add a little more info for the user here? Actually think this should come with a bit of a warning since it's going to start sending the password in plain text, so something like `Send password in plain text (make sure your connection is secure!)` for on and `Send password as token + salt` for off.